### PR TITLE
Logger refactor

### PR DESCRIPTION
**Pull request by danielcNet-bet**
This PR is not final , I am not going to merge it to main
Only for verifying that logger is Ok

### CODE REVIEW

File: core/logger2/logger.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
+package logger2
```
Comment (kolbis) [inline core/logger2/logger.go:1]: when you merge change the package name to logger, logger_test

File: core/logger2/logger.go [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+// Log implements Logger by calling logger.kitLogger.Log
+// If logger.IsExpired return true -> logger will reload himselfe
+func (log logger) Log(keyvals ...interface{}) error {
+	if log.IsExpired() {
+		log.Reload()
+	}
+	return log.kitLogger.Log(keyvals...)
+}
+
+// IsExpired  return true if day of logger.dateCreated and time.Now are not eq
```
Comment (kolbis) [inline core/logger2/logger.go:33]: documentation is not complete

File: core/logger2/logger.go [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+func (log logger) IsExpired() bool {
+	return log.dateUpdated.Day() != time.Now().UTC().Day()
+}
+
+// Reload function create new log.Logger object
```
Comment (kolbis) [inline core/logger2/logger.go:38]: question: why is this needed?

**Review by kolbis — CHANGES_REQUESTED** [review]

File: core/logger2/logger.go (hunk 3 above)
Comment (danielcNet-bet) [reply core/logger2/logger.go:38]: to verify if we need to create new file

File: core/logger2/logger.go (hunk 1 above)
Comment (danielcNet-bet) [reply core/logger2/logger.go:1]: as I wrote in PR comments
I am not going to merge it to main now , after completing review I am going to remove old logger and change implementation of logger in other packages and only after that -> merge to main

Comment (kolbis) [reply core/logger2/logger.go]: when you merge to main make sure you rename the package.

File: core/logger2/logger.go (hunk 3 above)
Comment (kolbis) [reply core/logger2/logger.go]: IsExpire is a bad name to indicate this.

**Review by kolbis — APPROVED** [review]

**Pull request merged**